Sources/StreamChatUI/ChatMessageList/ChatMessageListTitleView.swift

```diff
@@ -5,43 +5,41 @@
 import UIKit
 import StreamChat
 
-public class ChatMessageListTitleView<ExtraData: ExtraDataTypes>: UIView, UIConfigProvider {
-    public var title: String? {
-        get { titleLabel.text }
-        set { titleLabel.text = newValue }
+open class _ChatMessageListTitleView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
+    open var content: (title: String?, subtitle: String?) = (nil, nil) {
+        didSet {
+            updateContent()
+        }
     }
     
-    public var subtitle: String? {
-        get { subtitleLabel.text }
-        set { subtitleLabel.text = newValue }
-    }
-    
-    private weak var titleLabel: UILabel!
-    private weak var subtitleLabel: UILabel!
+    open private(set) var titleLabel: UILabel = UILabel()
+    open private(set) var subtitleLabel: UILabel = UILabel()
     
-    public init() {
-        super.init(frame: .zero)
+    public override func defaultAppearance() {
+        super.defaultAppearance()
         
-        let titleLabel = UILabel()
         titleLabel.textAlignment = .center
         titleLabel.font = uiConfig.font.headlineBold
-        self.titleLabel = titleLabel
-
-        let subtitleLabel = UILabel()
+        
         subtitleLabel.textAlignment = .center
         subtitleLabel.font = uiConfig.font.caption1
         subtitleLabel.textColor = uiConfig.colorPalette.subtitleText
-        self.subtitleLabel = subtitleLabel
-
+    }
+    
+    open override func setUpLayout() {
+        super.setUpLayout()
+        
         let titleView = UIStackView(arrangedSubviews: [titleLabel, subtitleLabel])
         titleView.axis = .vertical
         addSubview(titleView)
         titleView.translatesAutoresizingMaskIntoConstraints = false
         titleView.pin(to: self)
```

Comment:
I think it would be beneficial to have the stack view as a variable so it's easier to tweak it when you subclass this.

Reply:
👍